### ISSUE TITLE
1.17.0 containers not available through gcr.io/istio-release

### ISSUE THREAD

**Issue by tolleiv** [istio/istio#43556]
### Bug Description

The  gcr.io/istio-release repository is documented as official mirror for container images. Currently it lacks the v1.17.0 release containers.
Even though images can still be retrieved through hub.docker.com it's a bit inconvenient for users to switch between repositories.

Would be great to have all versions available on gcr.io/istio-release after the official release.

### Version

```prose
$ istioctl version
client version: 1.17.0
control plane version: 1.16.2
```

**Comment by zirain** [comment]
cc @istio/release-managers-1-17

**Comment by howardjohn** [comment]
1.17.1 is out now and fixes this. Sorry we decided we couldn't safely fix the botched 1.17.0 release. Good news is 1.17.1 is even better :wink:
